addons/web/static/src/core/position/utils.js

```diff
@@ -164,7 +164,7 @@ function computePosition(popper, target, { container, flip, margin, position, sh
     function getPositioningData(d, v) {
```

Comment:
This function seems relatively central, as it's used by "reposition" which is in turn used by "many2manyattendeeexpandable", properties fields, and reposition_hook. This last one then leads to a whole lot of other usages :smile: 

I'm not the best person to review these changes, should we ask the framework team? They usually ask for a test but in this specific case as it's a positioning matter, I'm not sure how easy it is to test :thinking: 

A workaround would be to use a different positioning for the component we are trying to "fix", not sure if that is an option.

Reply:
Hello @amdi-odoo this looks good but, unfortunately, I think you can still write a unit test. There are a couple of tests inside popover.test.js you can take inspirations from, you can have a custom containers and have a large empty div to force your popover target to be almost outside the container, then validate the popover position